results/griggin__rody_emb/97df40457cc37d916f7020bb6f1b07c9ae4d5ba1/model_meta.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Seems like this model is not implemented in MTEB. It also contains no description of the hugging face. You can see the checklist for how to add it